kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: migrating-visibility-data-from-es6
+title: Migrating Visibility Data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+Support for Elasticsearch v7 (ES7) was added to the 1.7.0. Temporal Server update.
+This introduces several breaking changes, including the removal of mapping types.
+These changes make ES6 incompatible with ES7.
+
+There are several ways to update your Temporal Server to support ES7.
+Upgrade ES6 to ES7, and then use one of the methods below to update your server.
```

Comment:
```suggestion
You can choose from various ways to update Temporal Server to support ES7.
Upgrade ES6 to ES7, and then use one of the following methods to update your server.
```
First sentence: One possible way to rephrase for [active voice](https://developers.google.com/style/voice). As a general rule, a sentence that begins "There are" or "There is" is ripe for rewriting.)
Second sentence: Per [below](https://developers.google.com/style/word-list#below).